Sonic-06-Mod-Manager/src/Environment3/RushInterface.cs

```diff
@@ -2266,10 +2266,10 @@ private void InstallThread(bool mods, bool patches)
                     }
 
             // Encrypt if decrypted EBOOT
-            if (PatchEngine.decrypted && Literal.System(Properties.Settings.Default.Path_GameExecutable) == "PlayStation 3") {
-                Console.WriteLine($"[{DateTime.Now:hh:mm:ss tt}] [Patch] Encrypted game executable...");
-                PatchEngine.EncryptExecutable();
-            }
+            //if (PatchEngine.decrypted && Literal.System(Properties.Settings.Default.Path_GameExecutable) == "PlayStation 3") {
+              //  Console.WriteLine($"[{DateTime.Now:hh:mm:ss tt}] [Patch] Encrypted game executable...");
+            //    PatchEngine.EncryptExecutable();
+          //  }

```

Comment:
Don't comment out code that needs removing, just remove it - we have commit history for this.